src/blocks/pacman.rs

```diff
@@ -0,0 +1,53 @@
+use std::time::Duration;
+use std::process::Command;
+
+use block::Block;
+use widgets::text::TextWidget;
+use widget::I3BarWidget;
+
+use serde_json::Value;
+use uuid::Uuid;
+
+
+pub struct Pacman {
+    text: TextWidget,
+    id: String,
+    update_interval: Duration,
+}
+
+impl Pacman {
+    pub fn new(config: Value, theme: Value) -> Pacman {
+        {
+            Pacman {
+                id: Uuid::new_v4().simple().to_string(),
+                update_interval: Duration::new(get_u64_default!(config, "interval", 5), 0),
+                text: TextWidget::new(theme.clone()).with_text(""),
```

Comment:
If you don't add text right away, just omit the .with_text. Also, you should add an Icon :) Add it from the awesome fonts icon set to the icons.rs file, don't forget to put an empty default value into the no icons JSON, for people that don't plan to use an icon font. Then use your icon by calling .with_icon("<name of your icon>")